_ml-commons-plugin/api.md

```diff
@@ -167,25 +225,50 @@ OpenSearch responds with the `task_id` and task `status`.
 }
 ```
 
-To see the status of your model registration, enter the `task_id` in the [task API] ...
+To see the status of your model registration and retrieve the model ID created for the new model version, pass the `task_id` as a path parameter to the Tasks API:
+
+```json
+GET /_plugins/_ml/tasks/<task_id>
+```
+{% include copy-curl.html %}
+
+The response contains the model ID of the model version:
 
 ```json
 {
-  "model_id" : "WWQI44MBbzI2oUKAvNUt", 
-  "task_type" : "UPLOAD_MODEL",
-  "function_name" : "TEXT_EMBEDDING",
-  "state" : "REGISTERED",
-  "worker_node" : "KzONM8c8T4Od-NoUANQNGg",
-  "create_time" : 1665961344003,
-  "last_update_time" : 1665961373047,
-  "is_async" : true
+  "model_id": "Qr1YbogBYOqeeqR7sI9L",
+  "task_type": "DEPLOY_MODEL",
+  "function_name": "TEXT_EMBEDDING",
+  "state": "COMPLETED",
+  "worker_node": [
+    "N77RInqjTSq_UaLh1k0BUg"
+  ],
+  "create_time": 1685478486057,
+  "last_update_time": 1685478491090,
+  "is_async": true
 }
 ```
 
 ## Deploying a model
 
 The deploy model operation reads the model's chunks from the model index and then creates an instance of the model to cache into memory. This operation requires the `model_id`.
 
+### Model access control considerations
+
+For clusters with model access control enabled, the following users can deploy models in model groups with the specified access levels:
```

Comment:
This part seems same with register model API. Actually model access control will be applied to all APIs. Maybe we don't need to specifically mention the access control for register and deploy model API.